packages/fluentui/react-builder/src/components/List.tsx

```diff
@@ -37,6 +37,74 @@ export const List: React.FunctionComponent<ListProps> = ({ onDragStart, style })
     },
   );
 
+  const titleRenderer = (Component, { content, expanded, open, hasSubtree, ...restProps }) => (
+    <Component expanded={expanded} hasSubtree={hasSubtree} {...restProps}>
+      {hasSubtree ? expanded ? <TriangleDownIcon /> : <TriangleEndIcon /> : null}
+      {content}
+    </Component>
+  );
+
+  const componentGroupItems = {};
+  const addComponentToGroup = (info, property) => {
+    componentGroupItems[property].push({
+      id: info.displayName,
+      title: (
+        <Box
+          key={info.displayName}
+          onMouseDown={handleMouseDown(info)}
+          styles={{
+            padding: '0.25em 0.75em',
+            cursor: 'pointer',
+            ':hover': {
+              background: '#ddd',
+              borderLeft: '2px solid #000',
+            },
+            borderLeft: '2px solid transparent',
+            marginLeft: '2px',
+          }}
+        >
+          {displayMode === 'Rendered' && (
+            <div style={{ position: 'relative' }}>
+              <div style={{ fontWeight: 'bold', opacity: 0.5 }}>{info.displayName}</div>
+              <div style={{ padding: '0.5rem', pointerEvents: 'none' }}>
+                {/* FIXME {React.createElement(resolveComponent(info.displayName), resolveDraggingProps(info.displayName))} */}
+              </div>
+            </div>
+          )}
+
+          {displayMode === 'Display Name' && info.displayName}
+        </Box>
+      ),
+    });
+  };
+
+  const componentTreeItems = [];
+  for (const property in COMPONENT_GROUP) {
+    componentGroupItems[property] = [];
+    COMPONENT_GROUP[property].forEach(item =>
+      supportedComponents.forEach((o, index) => {
+        if (o.displayName === item || (o.isChild && o.parentDisplayName === item)) {
+          delete supportedComponents[index];
```

Comment:
Do we really need to delete it?

Reply:
I delete the items because latter on I add all the remaining items into the "Others" group. I could set it to some value like undefined, that might be better?